tests/baselines/reference/tsc/incremental/initial-build/noEmit-changes-composite.js

```diff
@@ -310,8 +310,11 @@ Output::
     'prop1' is declared here.
 
 
-Found 2 errors.
+Found 2 errors in 2 files.
 
+Errors   Files
+     1   /src/project/src/directUse.ts:2
+     1   /src/project/src/indirectUse.ts:2
 exitCode:: ExitStatus.DiagnosticsPresent_OutputsGenerated
 
```

Comment:
In comparison to the original I think we might need an extra newline _after_ the table to be consistent with our newlines 👍🏻

Reply:
Got it, I'll add an extra newline.